src/pages/en/guides/deploy/edgio.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Deploy your Astro Site to Edgio
+description: How to deploy your Astro site to the web using Edgio.
+layout: ~/layouts/DeployGuideLayout.astro
+i18nReady: true
+---
+
+You can deploy your Astro project to [Edgio](https://www.edg.io/), an edge and CDN platform to deploy, protect and accelerate websites and APIs.
+
+:::tip
+Check out [the Astro guide in Edgio’s docs](https://docs.edg.io/guides/astro)!
+:::
+
+## How to deploy
+
+1. Install [the Edgio CLI](https://docs.edg.io/guides/cli) globally from the Terminal, if you haven’t already.
+
+```bash
+npm install -g @edgio/cli
+```
+
+2. Add Edgio to your Astro site
+
+```bash
+edgio init
+```
+
+3. (Optional) Enable Server Side Rendering
+
+After you’ve setup [Server Side Rendering with Astro](/en/guides/server-side-rendering/), specify the server file path in `edgio.config.js` as below:
+
+```js ins={2,5-9}
+// edgio.config.js
+import { join } from 'path'
+
+module.exports = {
+  astro: {
+    // The path of the standalone server that runs Astro SSR.
+    // The dependencies for this file are automatically bundled.
+    appPath: join(process.cwd(), 'dist', 'server', 'entry.mjs'),
+  },
+};
+```
```

Comment:
````suggestion
    ```js ins={2,5-9}
    // edgio.config.js
    import { join } from 'path'

    module.exports = {
      astro: {
        // The path of the standalone server that runs Astro SSR.
        // The dependencies for this file are automatically bundled.
        appPath: join(process.cwd(), 'dist', 'server', 'entry.mjs'),
      },
    };
    ```
````